pyclam/tests/utils.py

```diff
@@ -1,17 +1,19 @@
 """ Utilities for Testing.
 """
+from typing import Dict
+
 import numpy as np
 from scipy.spatial.distance import cdist
 
 from pyclam.types import Data, Radius
 from pyclam.utils import BATCH_SIZE
 
 
-def linear_search(point: Data, radius: Radius, data: Data, metric: str):
+def linear_search(point: Data, radius: Radius, data: Data, metric: str) -> Dict[int, float]:
```

Comment:
Docstring? it's a test util so the bar is lower, but I still think we should document all of our public interfaces

Reply:
done